egs/lre07/v1/lid/extract_ivectors_dnn.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# Copyright     2013  Daniel Povey
+#          2014-2015  David Snyder
+#               2015  Johns Hopkins University (Author: Daniel Garcia-Romero)
+#               2015  Johns Hopkins University (Author: Daniel Povey)
+# Apache 2.0.
+
+# This script extracts iVectors for a set of utterances, given
+# features and a trained DNN-based iVector extractor.
+
+# Begin configuration section.
+nj=30
+cmd="run.pl"
+stage=0
+min_post=0.025 # Minimum posterior to use (posteriors below this are pruned out)
+posterior_scale=1.0 # This scale helps to control for successive features being highly
+                    # correlated.  E.g. try 0.1 or 0.3.
+# End configuration section.
+
+echo "$0 $@"  # Print the command line for logging
+
+if [ -f path.sh ]; then . ./path.sh; fi
+. parse_options.sh || exit 1;
+
+
+if [ $# != 5 ]; then
+  echo "Usage: $0 <extractor-dir> <data> <ivector-dir>"
+  echo " e.g.: $0 exp/extractor_2048_male data/train_male exp/ivectors_male"
```

Comment:
This needs to be updated. E.g., script has 5 arguments, but only 3 specified in usage message. And references to male/female don't make sense for LID.